tools/DatasetExtraction/test_chords_txt/1.txt

```diff
@@ -0,0 +1,62 @@
+0.000000 2.612267 N

```

Comment:
As far I understand these are your test files generated for checking correctness of the loading mechanism?
In this case we don't need them in the repo. Otherwise think through file/directory structure and come up with self explanatory naming. How's one supposed to know what dataset and song is represented by `1.txt`?